src/components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,115 @@
+import { gray } from "d3-color";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useEffect, useState } from "react";
```
see comment in search.js